### PR TITLE
correct doc for DF.var

### PR DESCRIPTION
**Pull request by Rubtsowa**

### CODE REVIEW

**Comment by AlexanderKalistratov** [comment]
Are you sure `ddof` is not supported?

**Comment by AlexanderKalistratov** [comment]
I'm 95% sure `skipna` is supported

File: sdc/datatypes/hpat_pandas_dataframe_functions.py [hunk 1]
```diff
@@ -836,32 +836,35 @@ def std_overload(df, axis=None, skipna=None, level=None, ddof=1, numeric_only=No
...
+    Intel Scalable Dataframe Compiler User Guide
+    ********************************************
 
-       .. only:: developer
+    Pandas API: pandas.DataFrame.var
 
-           Test: python -m sdc.runtests -k sdc.tests.test_dataframe.TestDataFrame.test_var*
+    Limitations
+    -----------
+    Parameters ``axis``, ``level``, ``ddof`` and ``numeric_only`` unsupported.
```
Comment (densmirn) [inline sdc/datatypes/hpat_pandas_dataframe_functions.py]:
```suggestion
    Parameters ``axis``, ``level`` and ``numeric_only`` are supported only with default value ``None``.
```

File: sdc/datatypes/hpat_pandas_dataframe_functions.py [hunk 2]
```diff
@@ -827,7 +827,7 @@ def var_overload(df, axis=None, skipna=None, level=None, ddof=1, numeric_only=No
 
     Limitations
     -----------
-    Parameters ``axis``, ``level``, ``ddof`` and ``numeric_only`` unsupported.
+    Parameters ``axis``, ``level`` and ``numeric_only`` are supported only with default value ``None``.
```
Comment (densmirn) [inline sdc/datatypes/hpat_pandas_dataframe_functions.py]: According to `check_type()` parameters `axis`, `level` and `numeric_only` don't always support default value `None`. So you need to change either the docstring or the code in typing stage.

Comment (Rubtsowa) [reply sdc/datatypes/hpat_pandas_dataframe_functions.py]: @densmirn Parameters axis, level, numeric_only only None or Ommited

**Pull request merged**